api/pkg/utils/utils.go

```diff
@@ -215,3 +215,13 @@ func GetTimeoutSec(objSize int64) int64 {
 
 	return tmoutSec
 }
+
+// Function to check if the slice contains a string
+func SliceContainsString(inp string, list []string) bool {
```

Comment:
May be function can be renamed. ex: ObjectTyeStorageCheck(), this will give fair idea, during readability of code